File: source/EOS_gml/extensions/ExtensionCore/AndroidSource/Java/GMExtUtils.java
package ${YYAndroidPackageName};
import com.yoyogames.runner.RunnerJNILib;

public class GMExtUtils 
{
	public static String GetExtensionOption(String extName, String optName) {
		return RunnerJNILib.extOptGetString(extName, optName);
	}
}

File: source/EOS_gml/extensions/GMInstallSource/source/code_gen/core/GMExtUtils.swift
import Foundation

@objcMembers
public final class GMExtUtils: NSObject {

    /// Swift-facing wrapper that mirrors the C++ API shape:
    ///   gm.ExtUtils.getExtensionOption(...)
    public static func getExtensionOption(_ extName: String,
                                            optionName optName: String) -> String? {
        GMExtUtilsBridge.getExtensionOption(extName, optionName: optName)
    }
}
